packages/generator/test/download-ramls-gulp-task-helpers.test.ts

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Test file names should be camel case.

Reply:
I don't know what the rationale behind naming them kebab-case is but I would like to know that before changing the names. And that should be a separate PR.

Reply:
We started with the more widely accepted practice of kebab-case, but changed to align with the LWC. It's what we call out in our best practices now. We didn't make it high priority to rename tests, but decided to just update them as we go.

Reply:
Thanks for clarifying @bredmond-sf, will change them.